### DISCUSSION TITLE
Account activity transfer

### DISCUSSION THREAD

**Discussion by glchetan** [community/community#66441] [New to GitHub]
I'd like to consolidate the activity from my secondary GitHub account into my personal GitHub account, which already has access to numerous organizations. Since I can't modify the email associated with my personal GitHub account, what's the best way to merge the activity of secondary account into my personal GitHub account?

**Comment by queenofcorgis** [comment]
@glchetan - are you able to add your secondary email to your primary GitHub account?

**Comment by glchetan** [comment]
Yes, I can.

but what will be the benefit of that?

**Reply by queenofcorgis** [reply]
If you're trying to merge your contributions / activity to your primary account - this will do so. If that is not what you're intending to do would you mind sharing a few more details so the community can better assist?

**Comment by glchetan** [comment]
so, what you are suggesting that I should add the same email attached to my
other github account? to my primary github account ?

**Comment by glchetan** [comment]
when I tried adding that email I got the below error message.

**Comment by frg** [comment]
If you're referring to the activity/contribution section on your GitHub profile, unfortunately there isn't a way to migrate this onto another GitHub profile.

Does this answer your question @glchetan ?

![image](https://github.com/community/community/assets/1245426/a1818708-d82c-487c-9f1a-7ee77348a8c6)
